ts/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@computer-graphics-course/scene-format",
+  "description": "scene format for computer graphics course",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
I thought we need typescript to build something. Or do we send an already built package to npm?

Reply:
Added it to dev dependencies, thanks.